helm/hpcc/values.schema.json

```diff
@@ -487,9 +487,13 @@
           "type": "string",
           "description": "The amount of overall resourced memory to dedicate to the query"
         },
-        "thirdParties": {
+        "thirdParty": {
           "type": "string",
           "description": "The amount of overall resource memory to reserve for 3rd party use"
+        },
+        "maxMemPercentage": {
+          "type": "string",
```

Comment:
number?

Reply:
I wanted it as a real, but that is not a json schema supported type.

Reply:
I think "number" represents a floating point type, while "integer" does not allow a fractional part.

Reply:
you're right, have changed to "number"